File: 100 Days Coding Part-1/32. Toggle/toggle_char.py
#                                          Day 32 coding Statement 

# Write a Python Program to Toggle each character in a string

# Description
#  Get an input string from user and then convert the lower case of alphabets to upper case and 
#  all upper-case alphabets into lower case.

# Input
#  Enter a String : Hello
# Output
#  Toggle String : hELLO


# Method 1 :→ 

# str = input("Enter a String: ")
# str1 = ""        # initialize an empty string
# for i in str:
#     if i.isupper():
#         i = i.lower()
#         str1 = str1 + i
#     else:
#         i = i.upper()
#         str1 = str1 + i
# print("Toggle String : ",str1)        

# Method 2 :→ 

str = input("Enter a String: ")
print("Toggle String : ", str.swapcase())

File: 100 Days Coding Part-1/32. Toggle/toggle_char.java
//                                          Day 32 coding Statement 

// Write a Java Program to Toggle each character in a string

// Description
//  Get an input string from user and then convert the lower case of alphabets to upper case and 
//  all upper-case alphabets into lower case.

// Input
//  Enter a String : Hello
// Output
//  Toggle String : hELLO

import java.util.Scanner;
public class toggle_char{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in); 
        System.out.print("Enter a String : ");
        String str = sc.nextLine();                                           // Hello
        String str1 = "";
        for(int i = 0; i < str.length(); i++)
        {
            if(Character.isUpperCase(str.charAt(i)))                       
                str1 = str1 + Character.toLowerCase(str.charAt(i));        // hELLO
            else
                str1 = str1 + Character.toUpperCase(str.charAt(i));
        }
        System.out.println("Toggle String : "+str1);
    }
}